packages/material/test/renderers/MaterialArrayLayout.test.tsx

```diff
@@ -431,4 +441,54 @@ describe('Material array layout', () => {
       .find({ 'aria-label': 'Move down' });
     expect(downButton.is('[disabled]')).toBe(true);
   });
+
+  it('should render first simple property as child label', () => {
+    const store = initJsonFormsStore();
+    store.dispatch(Actions.init(data, schema, uischemaWithSortOption));
+    wrapper = mount(
+        <Provider store={store}>
+          <JsonFormsReduxContext>
+            <MaterialArrayLayout
+                schema={schema}
+                uischema={uischemaWithSortOption}
+            />
+          </JsonFormsReduxContext>
+        </Provider>
+    );
+    const childLabel1 = wrapper
+        .find('.MuiGrid-root.MuiGrid-item.MuiGrid-grid-xs-10.MuiGrid-grid-md-11')
```

Comment:
Should we add a class to that element so it can be selected easier?

Reply:
This question made me realize that we have no proper semantics defined for this label. I now introduced `aria-labelledby` to add respective semantics.